client/packages/components/src/components/notifications/components/NotificationWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, PropsWithChildren, useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { NotificationCard } from './NotificationCard';
+import { Notification } from '../types/Notification';
+
+const timesMilliSeconds = 100;
+const StyledMessageWrapper = styled.div<{ delay: number }>`
```

Comment:
Correct with PascalCase here?

Reply:
Yes this is a Component ant components a t pascal